.bazelrc

```diff
@@ -33,6 +33,9 @@ build --experimental_strict_action_env
 # in us getting a certificate for githubassets.com instead.
 startup --host_jvm_args=-Xms1g --host_jvm_args=-Xmx1g --host_jvm_args=-Djdk.tls.client.protocols=TLSv1.2
 
+# Enable sandboxing and caching for exclusive tests
```

Comment:
this is the equivalent of the patch we had before. Nice to see this finally upstreamed.